client/lang/src/en.json

```diff
@@ -9,5 +9,14 @@
     "MFAWebAuthnRegister.REGISTERING": "Registering",
     "MFAWebAuthnRegister.RETRY": "Retry",
     "MFAWebAuthnRegister.SUCCESS": "Key verified",
-    "MFAWebAuthnRegister.WAITING": "Waiting"
+    "MFAWebAuthnRegister.WAITING": "Waiting",
+
+    "MFAWebAuthnVerify.DESCRIPTION": "Use your secuirty key to verify your identity",
+    "MFAWebAuthnVerify.FAILURE": "Something went wrong, Please re-insert your key and try again",
+    "MFAWebAuthnVerify.HELP": "How to use security keys.",
+    "MFAWebAuthnVerify.INSTRUCTION": "Please insert your security key and {button}. ",
+    "MFAWebAuthnVerify.RETRY": "Try again",
+    "MFAWebAuthnVerify.SUCCESS": "Logging in…",
```

Comment:
```suggestion
    "MFAWebAuthnVerify.SUCCESS": "Logging in...",
```

Reply:
Semantically this is a bit of a pet peeve. `&hellip;` didn't work so I just copied it in :) I'll reset, it'll be closer to the design.

Reply:
Is there a problem with typing three dots?

Reply:
No idea, haha. "Pet peeves" being what they are.